example/readFromString/readFromString.cpp

```diff
@@ -0,0 +1,33 @@
+#include "json/json.h"
+#include <iostream>
+/*
+parse a string to Value object with CharReaderBuilder class or Reader class
```

Comment:
The CharReaderBuilder class cannot parse objects. I would rephrase as:

"Parse a raw string into a Value object using the CharReader class, or the legacy Reader class."